src/GeoPing.FrontEnd/src/DTO/types/stateTypes/allUsersFilterStateType.ts

```diff
@@ -0,0 +1,18 @@
+import * as moment from 'moment';
+
+export interface IStatusUsers {
+  name: string;
+  id: number;
+}
+
+export default interface IAllUsersFilterStateType {
+  name: string;
+  startDate: moment.Moment;
+  endDate: moment.Moment;
+  listCountFrom: string;
+  listCountTo: string;
+  status: Array<IStatusUsers>;
```

Comment:
maybe "UserStatus"  ?

Reply:
maybe, I'll fix it when I have all the code in the heap